README.md

```diff
@@ -1,70 +1,61 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+[![Netlify Status](https://api.netlify.com/api/v1/badges/e0f14414-e46b-4264-85f4-02a55abbb7e4/deploy-status)](https://app.netlify.com/sites/remarkable-cupcake-36a101/deploys)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Crypto Bazaar
 
-## Available Scripts
+This is a React app that uses the [Coin-Cap API](https://docs.coincap.io/). Displays over a 1000 crypto currencies, their current values and graph of their changes over the recent days.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- [React](https://reactjs.org/)
+- [Redux](https://redux.js.org/)
+- [React-Router](https://reactrouter.com/)
+- Chart.js
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Live Demo Link](https://cryptobazaar.netlify.app/)
 
-### `npm test`
+## Getting Started
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+To get a local copy up and running follow these simple example steps.
 
-### `npm run build`
+- Copy this link `https://github.com/nottherealalanturing/cryptobazaar`.
+- Get the directory that you want to clone the repository.
+- Open the command prompt in this directory.
+- Write `https://github.com/nottherealalanturing/cryptobazaar.git`.
+- Go to the repository folder in your command prompt `cd cryptobazaar`.
+- Run `npm install` to install the React and linter dependencies.
+- Run `npm build` to build the application with react .
+- Run `npm run start` to start the development server.
+- Run `npm run test` to run the tests.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.